tests/stage-test/test_staging.sh

```diff
@@ -3,12 +3,16 @@ set -euo pipefail
 
 # Integration test to make sure staging is working properly
 
-export CEPH_VERSION=luminous
+export CEPH_VERSION_SPEC=luminous-12.2.1-0
+# shellcheck disable=SC2155
+export CEPH_VERSION=$(maint-lib/ceph_version.sh "${CEPH_VERSION_SPEC}")
```

Comment:
I don't feel comfortable about ignoring SC2155 here. If you are already adding the shellcheck disable line, it is no more verbose to set CEPH_VERSION on one line and then export it after.

Reply:
It is for code safety that SC2155 throws an error. Ignoring it risks errors on corner cases.

Reply:
why not. doesn't make a big change.